### ISSUE TITLE
Simplify UI regarding access to metadata editing / renaming of recordings

### ISSUE THREAD

**Issue by soletsgit** [elan-ev/studip-opencast-plugin#964]
### tl;dr
To edit the Metadata of already recorded meetings simplify the access:
- add another button to the recordings entry with a link to the OC-Editor Metadata tab XOR
- add a "burger menu" (like the meetings-plugin-room-menue") to guide the user OC-Editor Metadata tab XOR 
- add a direct editing feature within the Stud.IP-OC-Plugin
- rename the "Open cutting tool" to something more generic like "Edit"

### Story
While using the current plugin version, I was wondering where to edit the metadata.
Recording of a Meeting-Room are usually named after the Meetings-Room.
But if these rooms have a generic name, the recordings have a corresponding name - which is often not specific enough.

### Challenge
- I just didn't find the "option" as it wasn't clear to me that the metadata editing is part of the cutting editor
- So please ease the access to metadata editing / renaming of recordings

**Comment by tgloeggl** [comment]
In V3 of the plugin, it is possible to edit the metadata for every video. :) 
I will keep this issue open for V2, but i do not think that this feature will be backported, since it is planned to switch to V3 in the next months.

**Comment by tgloeggl** [comment]
See also: https://github.com/elan-ev/studip-opencast-plugin/issues/504
